src/main/kotlin/app/revanced/patches/youtubevanced/ad/general/fingerprints/ContainsAdFingerprintSecondary.kt

```diff
@@ -0,0 +1,25 @@
+package app.revanced.patches.youtubevanced.ad.general.fingerprints
+
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import org.jf.dexlib2.AccessFlags
+import org.jf.dexlib2.Opcode
+
+object ContainsAdFingerprintSecondary:MethodFingerprint(
```

Comment:
The fingerprints seem very similar to me. They are both constrained to a method with the same name, with differences in parameters. Wouldn't it be sufficient to include the opcode pattern, the parameters, and the custom fingerprint field?